README.md

```diff
@@ -1,6 +1,8 @@
 # backup-cli
 Bash cli with various backup functionalities
 
+Please start with updating the config.template file.
```

Comment:
@unnisathyarajan config.template should be copied into `config` file, where users should update the details.